### PR TITLE
Container 43

### PR DESCRIPTION
**Pull request by pdietl**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -62,27 +63,26 @@ RUN yum update -y && yum install -y \
     # Add hmaccalc for generating FIPS hmac files
     hmaccalc \
     # Add SELinux policy devel
-    selinux-policy-devel && \
+    selinux-policy-devel \
+    # For running as a non-root user but being able to up privileges
+    sudo \
+    # Because VIM!
```
[the remark this answers is not in the document]
Comment (pdietl) [reply Dockerfile:69]: Dude it adds 2 MB

File: Dockerfile [hunk 2]
```diff
@@ -92,10 +92,26 @@ RUN pip install --upgrade pip && \
 # Install ronn for generating man pages
 RUN gem install ronn
 
-COPY dracut.conf /etc/dracut.conf
```
[the remark this answers is not in the document]
Comment (pdietl) [reply Dockerfile:95]: will add back. That was not intentional

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (eslerd) [reply Dockerfile:69]: ounces = lbs and lbs = pain

File: Dockerfile [hunk 3]
```diff
@@ -163,5 +170,51 @@ RUN yum erase -y cscope && \
...
-    make install && \
-    cd /tmp && rm -r cscope
+    make install DESTDIR=/tmp/cscope_install
+
+FROM main as sudoers
+COPY add_user_to_sudoers /tmp/add_user_to_sudoers
+RUN cd /tmp/add_user_to_sudoers && \
+    cargo build --release && \
+    mkdir -p /tmp/sudoers_install && \
+    install -m 4755 -t /tmp/sudoers_install target/release/add_user_to_sudoers
```
Comment (eslerd) [inline Dockerfile:180]: Did you want to clean this up?

Comment (pdietl) [reply Dockerfile:180]: why would I? I am totally abandoning this image except for the install dir

Comment (eslerd) [reply Dockerfile:180]: OK, nifty

**Pull request merged**